regulations/templates/regulations/comment-confirm-fail.html

```diff
@@ -0,0 +1,20 @@
+<div>
+  Comment submission failed.
+  We're sorry, something went wrong with our system, and submitting your comment did not work.
+</div>
+
+<div>
+  <a href="{% url 'comment_review' doc_number %}">Return to your draft comment</a> to try submitting again.
+</div>
+
+<div>
+  If this problem keeps happening
+  If you've already tried submitting your comment more than once, please save a copy of your draft comment for safekeeping.
```

Comment:
Sorry, I misunderstood that it was a header in the original document but lost that formatting when you copied it. Would you mind bringing it back? Looks like we used `h3`s in the previous copy.